prometheus.go

```diff
@@ -88,13 +88,7 @@ func Query(host string, start time.Time, end time.Time, query string) ([]Result,
 }
 
 func steps(dur time.Duration) int {
```

Comment:
Hm, this might result in very high amount data, for exporting data with 1w (or even longer) time ranges.